pre-docs/inventory_version2.md

```diff
@@ -90,6 +94,10 @@ groups:
     target-lookups:
       - plugin: puppetdb
         query: "inventory[certname] { facts.osfamily = 'RedHat' }"
+        config:
+          user: identity.user
```

Comment:
I think these got stale. Here is one i've been using
```yaml
version: 2
groups:
  - name: test
    target-lookups:
      - plugin: puppetdb
        query: inventory[certname]{facts.os.release.major = "7"}
        name: facts.ipaddress
        config:
          ssh:
            user: facts.identity.user
            host: facts.networking.hostname
    config:
      transport: ssh
      ssh:
        private-key: ~/.ssh/id_rsa-acceptance
```

Reply:
Updated!